include/fields2cover/swath_generator/swath_generator_base.h

```diff
@@ -21,6 +21,9 @@ namespace f2c::sg {
 
 template <typename T>
 class SwathGeneratorBase {
+ public:
+   bool allow_overlap {false};
```

Comment:
Hmm, I would say that overlap is a better choice here. Because it doesn't let the robot drive or cover anything out of the border of the field. Instead, it creates overlap between the last and one-before-last swath in such way that the last swath is exactly covering until the border of the field

Reply:
In the example above, the last swath was at 8.5, leaving a row of 3 uncovered (width of robot 5.5). With `allow_overlap`, it adds an additional swath at 5.5, so that the row of 3 is covered

Reply:
Okay, yes, you are right. Thanks for your contribution again!